pkg/domain/infra/abi/play.go

```diff
@@ -586,3 +586,73 @@ func getBuildFile(imageName string, cwd string) (string, error) {
 	}
 	return "", err
 }
+
+func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, path string, opts entities.PlayKubeOptions) (*entities.PlayKubeReport, error) {
+	var (
+		podNames []string
+	)
+	reports := new(entities.PlayKubeReport)
+
+	// read yaml document
+	content, err := ioutil.ReadFile(path)
```

Comment:
Performance nit:  With --down, Podman will read and parse the YAML twice (and also sent it twice over the wire for remote).

@baude, can you elaborate on why it's split into two separate calls?

Reply:
Correction: I missed the return on the cmd side and didn't notice that --down will return after teardown.